src/main/resources/portal.properties.EXAMPLE

```diff
@@ -204,12 +204,13 @@ ucsc.build=hg19
 oncoprint.defaultview=patient
 
 # OncoPrint driver mutation annotations
-# oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
-# oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
-# oncoprint.custom_driver_annotation.default=true
-# oncoprint.custom_driver_tiers_annotation.default=true
-# oncoprint.oncokb_hotspots.default=custom
-# oncoprint.hide_vus.default=false
+#   oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
+#   oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
+#   oncoprint.custom_driver_annotation.binary.default=true
```

Comment:
@jjgao @oplantalech I think we decided that we should not have defaults represented in configuration here but instead in our codebase.   it doesn't make a lot of sense to have defaults values in configuration, right?

Reply:
but this is the example file @alisman

Reply:
oh, now I understand what you are saying. @alisman , .default actually refers to a configuration - it is whether the binary, oncokb, etc selection are selected *by default*

Reply:
ok, that makes sense.  maybe we could have a better name.  i'll leave that up to you guys.